backend/infrahub/graphql/mutations/branch.py

```diff
@@ -242,53 +238,20 @@ class Arguments:
     object = Field(BranchType)
 
     @classmethod
-    @retry_db_transaction(name="branch_merge")
     async def mutate(cls, root: dict, info: GraphQLResolveInfo, data: BranchNameInput) -> Self:
         context: GraphqlContext = info.context
 
-        async with UserTask.from_graphql_context(title=f"Merge branch: {data['name']}", context=context) as task:
-            obj = await Branch.get_by_name(db=context.db, name=data["name"])
-
-            merger: Optional[BranchMerger] = None
-            async with lock.registry.global_graph_lock():
-                async with context.db.start_transaction() as db:
-                    merger = BranchMerger(db=db, source_branch=obj, service=context.service)
-                    await merger.merge()
-                    await merger.update_schema()
-
-            fields = await extract_fields(info.field_nodes[0].selection_set)
+        obj = await Branch.get_by_name(db=context.db, name=data["name"])
 
-            ok = True
+        if not context.service:
```

Comment:
For my information, might `context.service` be different from global `services.service` here?

Reply:
it can,
context.service is a different object passed down from the GraphQL App vs `services.service` which is a global variable